src/internal/connector/onedrive/collections.go

```diff
@@ -14,94 +14,89 @@ import (
 	"github.com/alcionai/corso/src/internal/observe"
 	"github.com/alcionai/corso/src/pkg/logger"
 	"github.com/alcionai/corso/src/pkg/path"
-	"github.com/alcionai/corso/src/pkg/selectors"
 )
 
-// Collections is used to retrieve OneDrive data for a
-// specified user
+type driveSource int
+
+const (
+	unknownDriveSource driveSource = iota
+	OneDriveSource
+	SharePointSource
+)
+
+type folderMatcher interface {
+	IsAny() bool
+	Matches(string) bool
+}
+
+// Collections is used to retrieve drive data for a
+// resource owner, which can be either a user or a sharepoint site.
 type Collections struct {
-	tenant string
-	user   string
-	scope  selectors.OneDriveScope
-	// collectionMap allows lookup of the data.Collection
-	// for a OneDrive folder
-	collectionMap map[string]data.Collection
+	Tenant        string
+	ResourceOwner string
+	Source        driveSource
```

Comment:
nit: I'd prefer to make these unexported if they aren't used outside of the package. If they're left exported other things may start using them and then it may become harder to make changes in the future

Reply:
Ah, good catch, I must have redacted the change that used these externally.